src/drivers/wayland/lv_wayland.c

```diff
@@ -1949,11 +1949,13 @@ static bool resize_window(struct window * window, int width, int height)
         width -= (2 * BORDER_SIZE);
         height -= (TITLE_BAR_HEIGHT + (2 * BORDER_SIZE));
     }
+    if(width <= 0 || height <= TITLE_BAR_HEIGHT) return false;
```

Comment:
It would be good to add parenthesis over the sub conditions, it would improve the readability and ditches away any issue with operator precedence